splice_ck/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This should be GNU Affero license, not ASLv2.0.  See our 3.0 spliceengine pom.xml.

Reply:
ok, I will fix it.